docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md

```diff
@@ -0,0 +1,310 @@
+| YBClient Latency              | Latency of local and remote read and write requests.<br>Refer to the YBClient Ops Local vs Remote description regarding local and remote requests. | This metric is informational and should not be subject to alerting. You may consider this information while examining alerts on other metrics. | ![YBClient Latency](/images/yp/metrics17.png) |
+| Reactor Delays                | The number of microseconds the incoming CQL requests spend in the worker queue before the beginning of processing. <br><br>Note that Reactor is a software implementation of a ring queue. | This value should be close to zero. If it is increasing or stays high, you should treat it as an indicator of a network issue or that the queues are full. If this is the case, you should investigate throughput and queue size and latency metrics for tuning guidance. | ![Reactor Delays](/images/yp/metrics18.png) |
+| RPC Queue Size                | The number of RPCs in the service queue.                     | The queue size is an indicator of incoming traffic and throughput. Typically, it either shows very low values, which means that YugabyteDB is processing requests at a healthy rate; or a flat line at the maximum-configured queue size, which means that YugabyteDB has filled the queues and cannot process fast enough. | ![RPC Queue Size](/images/yp/metrics19.png) |
+| Response Size (bytes)         | The size of the RPC response.                                | The response size for RPCs should be relatively small.<br>This metric is informational and should not be subject to alerting. You may consider this information while examining alerts on other metrics. | ![Response Size](/images/yp/metrics20.png) |
+| Transactions                   | The number of transactions.                                  | This value depends on the application or activity. Because transactions can have batched statements, it's not possible to provide a specific guidance for this metric. You may consider this information while examining alerts on other metrics. | ![Transaction](/images/yp/metrics21.png) |
+| Inbound RPC Connections Alive | The count of current connections at the CQL API level.       | If this spikes to a number much higher than your average, you should consider that there may be an active DDoS or a security incident. | ![Inbound RPC](/images/yp/metrics22.png) |
+
+### Tablet server
```

Comment:
Not sure if its possible , but can you reduce the width of the "Alert guidance" column so that the images for "Example" column appears tad bigger.

Reply:
Not really. I think these will go away eventually. But you can click them to display full size now.